deploy-manage/deploy/elastic-cloud/upload-custom-plugins-bundles.md

```diff
@@ -151,7 +151,13 @@ After uploading your files, you can select to enable them when creating a new {{
 
 While you can update the ZIP file for any plugin or bundle, these are downloaded and made available only when a node is started.
 
-You should be careful when updating an extension. If you update an existing extension with a new file, and if the file is broken for some reason, all the nodes could be in trouble, as a restart or move node could make even HA clusters non-available.
+:::{important}
+You should be careful when updating an extension. If you update an existing extension with a new file, and if the file is broken for some reason, all the nodes could be in trouble, as a restart or move node could make even HA clusters non-available. Also, shards of your indices may become unassigned there's anything wrong with the bundle, for example, file being referenced by index is missing due to the update. 
+:::
+
+:::{tip}
+If you need to make update to your extension, instead of updating an existing extension with a new file directly, we recommend you to create a new extension to test the behavior first, verify it's validty, and then reflect it to your deployment. 
```

Comment:
```suggestion
If you need to update your extension, instead of updating an existing extension with a new file directly, we recommend that you create a new extension to test the behavior first, verify it's validity, and then reflect it to your deployment.
```